wpilibc/Athena/src/PIDController.cpp

```diff
@@ -435,28 +410,28 @@ void PIDController::SetTolerance(float percent) {
 }
 
 /*
- * Set the percentage error which is considered tolerable for use with
+ * Set the absolute error which is considered tolerable for use with
```

Comment:
Did you actually change anything? or did u just accidentally flip the order of these two functions.